Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -0,0 +1,93 @@
+{-
+
+This file proves the higher groupoid structure of types
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.GroupoidLaws where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+
+-- some useful notation
+
+_·_ :  {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
+x≡y · y≡z = compPath x≡y y≡z 
+
+infixr 30 _·_
+
+_⁻¹ : {x y : A} → (x ≡ y) → (y ≡ x)
+x≡y ⁻¹ = sym x≡y
+
+𝟭 : {x : A} → x ≡ x
+𝟭 = refl
+
+-- homogeneous groupoid laws
+
+symInvo : {x y : A} (p : x ≡ y) →
+     Path (Path A x y) p ((p ⁻¹)⁻¹)
```

Comment:
Why not use `≡` instead of the explicit `Path`?

Reply:
Oh, how come I didn't think of it?